src/components/TransactionModal.tsx

```diff
@@ -42,9 +49,24 @@ const PendingTransactionModalBody = () => {
         </h2>
         <p>
           <Trans>
-            Your transaction has been submitted and is awaiting confirmation.
+            Your transaction has been submitted and is awaiting confirmation{' '}
           </Trans>
         </p>
+        <p>
+          {pendingTx ? (
+            <ExternalLink
```

Comment:
**suggestion** Could we use EtherscanLink here perhaps?

`src/components/EtherscanLink.tsx`

Reply:
Yeah, we can use `<EtherscanLink/>` but that component also returns an icon so it looks like this.
I've added children inside that component if `EtherscanLink` is a type of `tx`
![image](https://user-images.githubusercontent.com/18723426/198282766-8c111312-b3d7-4bd4-99da-ab0fac222508.png)

It looks cool to me, wdyt?

Reply:
We can put logic to remove the icon if there are children, but this looks good to me the way it is! :)